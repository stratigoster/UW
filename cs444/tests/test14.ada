PACKAGE foo IS
  s : STRING := ABS "Batman";
--              ^^^^^^^^^^^^^
-- abs takes a numeric type

  t : STRING := s MOD "B";
--              ^^^^^^^^^^
-- mod works on numeric values

  i,j,k,l : INTEGER := 1;
  m : INTEGER := i MOD j + ABS k - (3/2/l);
END foo;
