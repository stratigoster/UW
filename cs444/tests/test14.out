***ERROR***:2:19:	ABS takes a numeric type
***ERROR***:6:19:	MOD  works on numeric values of the same type

------
Compilation error
------
